src/main/resources/application.yml

```diff
@@ -7,10 +7,17 @@ management.server.port: 8082
 spring.data.rest.basePath: /
 
 spring.data.cassandra:
-  keyspace-name: spring_data_starter
+  keyspace-name: ${ASTRA_DB_KEYSPACE}
```

Comment:
I'm suggesting to use Spring Boot default properties as much as possible, this simplifies the amount of configuration code. The only property not handled by default is the secure connect bundle. I'm proposing to declare it below under a clearly distinctive prefix `datastax.astra`.